be/src/http/http_client.h

```diff
@@ -66,6 +66,11 @@ class HttpClient {
         curl_easy_setopt(_curl, CURLOPT_COPYPOSTFIELDS, post_body.c_str());
     }
 
+    void use_ssl() {
+        curl_easy_setopt(_curl, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
The third param should be a long type ??

Reply:
ok, I was careless